lib/csp.js

```diff
@@ -72,6 +73,12 @@ function addUpgradeUnsafeRequestsOptionTo (directives) {
   }
 }
 
+function addReportURI (directives) {
+  if (config.csp.reportURI) {
+    directives.reportURI = config.csp.reportURI
```

Comment:
Does this work? The [docs](https://helmetjs.github.io/docs/csp/) have `reportUri` and fields are case-sensitive, right?

I think it would make sense to use `reportUri` for the config option as well, for consistency with their format – that way we can't have confusions with their documentation.

Reply:
Agreed. But yes, it worked ^^